proposals/lotus-blockstore-interface.md

```diff
@@ -0,0 +1,44 @@
+# Extended Blockstore Interface
+
+Authors: vyzo
+
+## What is the problem this project solves?
+
+We currently have a limited "dumb" blockstore interface, which does not allow us to pass context options so as to inform the blockstore about certain properties of objects being stored.
+
+This is particularly important in the context of the splitstore, where we have the following problems:
+- We lack the ability to inform the splitstore of the actual epoch an object belongs, which requires us to "guess" by estimating based int he current tipset and wall clock (see [#6474](https://github.com/filecoin-project/lotus/pull/6474)). This is problematic for many reasons, as an object might be misassigned its epoch (and its completely broken during catch up sync in current master).
+- We lack the ability to inform the splitstore when an access is initiated from the network. In conjunction with the transactional gc logic in [#6474](https://github.com/filecoin-project/lotus/pull/6474) this would result in network requests keeping objects that would otherwise be pruged from the hotstore live.
```

Comment:
```suggestion
- We lack the ability to inform the splitstore when an access is initiated from the network. In conjunction with the transactional gc logic in [#6474](https://github.com/filecoin-project/lotus/pull/6474) this would result in network requests keeping alive objects that would otherwise be purged from the hotstore.
```

Reply:
The `ExposedBlockstore` abstraction is not sufficient.